### PR TITLE
Fix linters notifications

### PR DESCRIPTION
**Pull request by furdarius**
PR fixes some linters errors and warnings.

### CODE REVIEW

File: internal/ctxwatch/context_watcher_test.go [hunk 1]
```diff
@@ -87,6 +87,9 @@ func TestContextWatcherStress(t *testing.T) {
 		if i%2 == 1 {
 			cancel()
 		}
+
+		// To avoid context leak
+		cancel()
```
Comment (jackc) [inline internal/ctxwatch/context_watcher_test.go]: I don't think there is a context leak here. It is always canceled in one of the if statements. If i is even it is canceled before Unwatch and if it is odd it is canceled after the Unwatch.

Comment (furdarius) [reply internal/ctxwatch/context_watcher_test.go]: Just linter notifies about it, so I decided to make it more clear.
Should I revert this?

Comment (jackc) [reply internal/ctxwatch/context_watcher_test.go]: Yeah, I think so. It's annoying to have a linter warning, but I think that the logic of the test where half are canceled before and half canceled after is obscured by the superfluous cancel.

Comment (furdarius) [reply internal/ctxwatch/context_watcher_test.go]: Done)

**Comment by jackc** [comment]
Thanks!

**Pull request merged**